src/PeliasSearchManager.swift

```diff
@@ -213,31 +226,31 @@ public struct PeliasError {
 extension PeliasSearchResponse {
   
   //TODO: I'm still not sure of this approach - might be better to implement something more like a proper protocol like http://redqueencoder.com/property-lists-and-user-defaults-in-swift but this works for now
-  public class HelperClass: NSObject, NSCoding {
+  open class HelperClass: NSObject, NSCoding {
     
-    public var response: PeliasSearchResponse?
+    open var response: PeliasSearchResponse?
     
     public init(response: PeliasSearchResponse) {
       self.response = response
       super.init()
     }
     
     class func path() -> String {
-      let documentsPath = NSSearchPathForDirectoriesInDomains(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomainMask.UserDomainMask, true).first
-      let path = documentsPath?.stringByAppendingString("/Response")
-      return path!
+      let documentsPath = NSSearchPathForDirectoriesInDomains(FileManager.SearchPathDirectory.documentDirectory, FileManager.SearchPathDomainMask.userDomainMask, true).first
+      let path = (documentsPath)! + "/Response"
```

Comment:
Hmmm that's a bit of lazy code on my part, although to be honest I've never encountered `NSSearchPathForDirectoriesInDomains()` not returning something when you gave it the correct input values.

I'll have to change the method signature here to indicate it could return an optional though if we do the null value check, which will have ramifications at the call sites. Lemme take a look at those.